node/popn.go

```diff
@@ -755,6 +755,12 @@ func (nd *node) Load(ctx context.Context, args *GetArgs) (chan GetResult, error)
 
 			log.Info().Msg("selected an offer")
 
+			bal, err := nd.exch.Wallet().Balance(ctx, nd.exch.Wallet().DefaultAddress())
+			if err != nil {
+				log.Error().Err(err).Msg("error when fetching balance")
+			}
+			fmt.Printf("==> Fil balance: %s attofil\n", humanize.Comma(bal.Int64()))
```

Comment:
We have [utilities](https://github.com/myelnet/pop/blob/main/filecoin/types.go#L517) for doing that properly.